.travis.yml

```diff
@@ -7,4 +7,6 @@ branches:
 cache:
   directories:
   - node_modules/
-script: yarn prettier:check && yarn lint && yarn build:check
+script: yarn prettier:check && yarn lint && yarn build:check && yarn test && ./test/integration_tests.sh
+before_install:
+  - if [[ ! $(which expect) ]] ; then sudo apt-get install -y expect ; fi
```

Comment:
Replace this with an install via the `apt` addon: https://docs.travis-ci.com/user/installing-dependencies/#adding-apt-packages